CHANGES_NEXT_RELEASE

```diff
@@ -29,3 +29,4 @@
 - Fix: Trailing slashes in URI path no longer silently removed, but considered and giving adequate error responses
 - Fix: Avoid rendering at the same time id and idPattern fields in NGSIv2 subscriptions
 - Fix: Error returned on GET/DELETE requests specifying Content-Type (Issue #2128)
+- Fix: Error responses with '405 Method Not Allowed' now return payload to describe the error, for v2 requests only (Issue #2075)
```

Comment:
Ah, of course. I found about those after sending the PR. Didn't react about it :-)
Thanks!